CHANGELOG.md

```diff
@@ -1,3 +1,109 @@
+* Adds support for multiple DDL statements within a transaction
+
+* Adds support for certain foreign key constraints
+
+* Adds limited support for `INSERT INTO ... SELECT FROM` commands
+
+* Adds support for the `TRUNCATE` command
+
+* Adds support for `HAVING` clauses in `SELECT` queries
+
+* Adds support for `EXCLUDE` constraints which include the partition column
+
+* Adds support for system columns in queries (`tableoid`, `ctid`, etc.)
+
+* Adds support for relation name extension within `INDEX` definitions
+
+* Adds support for no-op `UPDATE`s of the partition column
+
+* Adds several general-purpose utility UDFs to aid in Citus maintenance
+
+* Adds `master_expire_table_cache` UDF to forcibly expire cached shards
+
+* Extends the router planner to handle more queries
+
+* Parallelizes the processing of DDL commands which affect distributed tables
+
+* Adds support for repartition jobs using composite or custom types
+
+* Enhances object name extension to handle long names and large shard counts
+
+* Parallelizes the `master_modify_multiple_shards` UDF
+
+* Changes distributed table creation to error if the target table is not empty
+
+* Changes the `pg_dist_shard.logicalrelid` column from an `oid` to `regclass`
+
+* Adds a true `placementid` column to `pg_dist_shard_placement`
+
+* Removes the `pg_dist_shard.shardalias` distribution metadata column
+
+* Adds `pg_dist_partition.repmodel` to track tables using streaming replication
+
+* Adds internal infrastructure to take snapshots of distribution metadata
+
+* Addresses the need to invalidate prepared statements on metadata changes
+
+* Adds a `mark_tables_colocated` UDF for denoting pre-6.0 manual colocation
+
+* Fixes a bug affecting prepared statement execution within PL/pgSQL
+
+* Fixes a bug affecting `COPY` commands using composite types
+
+* Separates worker and master job temporary folders
+
+* Eliminates race condition between distributed modification and repair
+
+* Relaxes the requirement that shard repairs also repair colocated shards
+
+* TODO: What about #785 ?
+
+* TODO: #840
+
+* TODO: #865
+
+* TODO: #864
+
+* Adds a `create_distributed_table` UDF (#867)
+
+* Adds a `create_reference_table` UDF (#892)
+
+### citus v5.2.2 (November 4, 2016) ###
+
+* Adds support for `IF NOT EXISTS` clause of `CREATE INDEX` command
+
+* Adds support for `RETURN QUERY` and `FOR ... IN` PL/pgSQL features
+
+* Extends the router planner to handle more queries
+
+* Changes `COUNT` of zero-row sets to return `0` rather than an empty result
+
+* Reduces the minimum permitted `task_tracker_delay` to a single millisecond
+
+* Fixes a bug that caused crashes during joins with a `WHERE false` clause
+
+* Fixes a bug triggered by unique violation errors raised in long transactions
+
+* Fixes a bug resulting in multiple registration of transaction callbacks
+
+* Fixes a bug which could result in stale reads of distribution metadata
+
+* TODO: What about #752 ? That's part of an unready feature, right?
```

Comment:
Nope, that's not part of Insert/Select, if thats what you mean. Its basically if you define a function which performs Insert statements. So we should document as bug-fix